### DISCUSSION TITLE
How to create the Local Environment while running GPT-Pilot in Docker Container

### DISCUSSION THREAD

**Discussion by brown7477** [Pythagora-io/gpt-pilot#273] [Q&A]
I have been testing GPT-Pilot for several days and one of the problems I am running into is when GPT-Pilot asks me to Create the Local Environment I am not sure how.
`"Node.js", "MySQL", "PeeWee", "ReactJS", "Socket.io", "Bootstrap", "Vanilla Javascript", "Cronjob"`
Can anyone help me out with how I do this? In docker and outside of Docker.
I have noticed if this task is not accomplished, GPT-Pilot will not and does not create the full code stack for the new app.

**Comment by YosuCadilla** [comment]
GPTP and GPT4 are great at coding but have difficulty with setting environments, even for the most common scenarios.
I guess in the future this will change, but as of now, you will have to install and configure many of these tools by yourself before you are able to proceed forward.
